scripts/build/clean.js

```diff
@@ -0,0 +1,45 @@
+
+const {exec} = require('child_process');
+const fs = require('fs');
+
+const {sync: globSync} = require('glob');
+
+function main() {
+  removeDirectory('build');
+  removeDirectory('.rewrite-tmp');
+  removeFilesOfType('js');
+  removeFilesOfType('d.ts');
+  removeFilesOfType('map');
+}
+
+function removeDirectory(directory) {
+  exec(`rm -rf ${directory}`, (err) => {
+    if (err) {
+      throw err;
+    }
+
+    console.log(`deleted ${directory} directory`);
+  });
+}
+
+function removeFilesOfType(type) {
+  const fileGlob = `packages/*!(node_modules|dist)/*.${type}`;
+  const nestedFileGlob = `packages/*/!(node_modules|dist)/*.${type}`;
```

Comment:
`packages/*/!(node_modules|dist)/**/*.${type}` would future-proof this in case we ever end up with more than one level of subdirectories for some reason.